geniesp/bpc_redcap_export_mapping.py

```diff
@@ -1962,7 +1964,7 @@ def run(self):
         treatment_data = self.get_timeline_treatment(
             df_map=redcap_to_cbiomappingdf, df_file=data_tablesdf
         )
-        if self._SPONSORED_PROJECT not in ["BrCa", "CRC"]:
+        if self._SPONSORED_PROJECT not in ["BrCa", "NSCLC", "NSCLC2"]:
```

Comment:
Do not merge until mapping file is fixed.  These lines of code can then be removed